todos/src/supabaseData.js

```diff
@@ -0,0 +1,90 @@
+import SupabaseSingleton from "supabase";
+import { WENT_WRONG_MESSAGE } from "constants";
+import { dangerMessage } from "toastMethods";
+
+const supabase = new SupabaseSingleton();
+export default supabase;
```

Comment:
We can remove it from here as we can export supbase instance from [todos/src/supabase.js](https://github.com/fza-wtag/todo-app-react/pull/12/files#diff-8a1d9cfd55b9ed6db76eda1ffcd8d0c884220b3a2dc445a4b5858c258b83e714)